### PR TITLE
Issue #49 Resolved (Footer)

### LINKED ISSUE #49: [Bug] <Footer Social Icons are all same and not redirecting>
### Description

Social icons in Footer sections are all same and they are also not getting redirected anywhere.

### Screenshots

Screenshot (65)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Jai-Solania-29**
## Fixes Issue

Closes #49 

## Changes proposed
**Social Icons** in the footer are now accurate and according to the design of the website.
**Redirection links** are given to the social icons, and they all are in working state following the target blank property.
**Footer Size** is modified according to the official design of the website.
Alignment of text and sponsor card fixed.
**Sponsor Card** size is modified according to the official design of the website.
**Design of Sponsor Card** is also modified and its now accurate and according to the design o the website.
**Font Size** is modified according to the design of the website.

## Screenshots
**Previous Footer:**

Screenshot (65)

**Current Footer :**

Screenshot (68)

## Note to reviewers
**cc**- @AvineshTripathi  @verma-kunal @SuperAayush 

Previously **SVG** format images were used for social icons, but they were not working properly as for all social icons same and only one image was rendering which was in the front line of the code.
So for fixing that issue I have now used **React Icons** instead of **SVG** files.

### CODE REVIEW

**Review by AvineshTripathi — COMMENTED** [review]
After trying the changes I see the overall changes alignment of the footer component is disturbed on resizing the screen please have a look into it 

Also we need some space between the mail logo and the mail Id

File: src/theme/Footer/index.js [hunk 1]
```diff
@@ -72,16 +81,22 @@ function Footer() {
...
       <div className={styles.mail_title}>example@kubesimplify.com</div>
       <div className={styles.group_logo}>
-        <a href="#"><logo.instagram.Svg row="img" className={styles.social}/></a>
+        <a target="_blank" href="https://www.youtube.com/c/saiyam911"><AiFillYoutube row="img" className={styles.social1}/></a>
+        <a target="_blank" href="https://discord.gg/eEEFPVMr"><FaDiscord row="img" className={styles.social2}/></a>
+        <a target="_blank" href="https://twitter.com/kubesimplify"><AiOutlineTwitter row="img" className={styles.social2}/></a>
+        <a target="_blank" href="https://www.linkedin.com/company/kubesimplify/"><TiSocialLinkedin row="img" className={styles.social2}/></a>
+        <a target="_blank" href="https://www.instagram.com/saiyampathak/"><AiOutlineInstagram row="img" className={styles.social2}/></a>
+        <a target="_blank" href="https://github.com/kubesimplify"><AiFillGithub row="img" className={styles.social3}/></a>
+        {/* <a href="#"><logo.instagram.Svg row="img" className={styles.social} /></a>
```
[the remark this answers is not in the document]
Comment (Jai-Solania-29) [reply src/theme/Footer/index.js:93]: I have removed these lines now, and they will reflect in my next commit.

**Comment by AvineshTripathi** [comment]
Hey @Jai-Solania-29 any updates

**Comment by Jai-Solania-29** [comment]
Hey @AvineshTripathi actually I'm having my exams these days so I'm little busy right now. I was thinking may be you can merge this PR for now, and I'll open a different issue for responsiveness of the footer so that someone else can work on that if interested, Otherwise I'll start working on this issue in next couple of days.

**Comment by AvineshTripathi** [comment]
All the best @Jai-Solania-29 for exams and thanks for notifying 

LGTM 

P.S. i ll create an issue for it after merge

**Comment by Jai-Solania-29** [comment]
Thank you @AvineshTripathi , I have my last exam tomorrow. So once you create an issue I’ll start working on that.

**Comment by Jai-Solania-29** [comment]
Hey @AvineshTripathi any updates ?

**Comment by AvineshTripathi** [comment]
Hey @Jai-Solania-29  sorry for the delay, can you create issue and start working or i should create issue?

**Comment by Jai-Solania-29** [comment]
Hey @AvineshTripathi I’ll create an issue and will start working on that , but that is only possible when my PR gets merged. I mean the actual footer will reflect after my PR gets merged, and only then I can open an issue for making the footer responsive, and will start working on that .

**Comment by AvineshTripathi** [comment]
LGTM with a reminder to myself the package.lock.json file has updated, in case tree gets disturbed after merge 

cc @verma-kunal

**Comment by verma-kunal** [comment]
> can you make the positioning relative and not absolute 

I believe this change hasn't been done yet! @Jai-Solania-29 can you pls check this one :)

**Comment by Jai-Solania-29** [comment]
> > can you make the positioning relative and not absolute
> 
> I believe this change hasn't been done yet! @Jai-Solania-29 can you pls check this one :)

Hey @verma-kunal I have pushed the requested changes, Kindly check and merge.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by verma-kunal** [comment]
Thanks for contributing @Jai-Solania-29 ✨

**Pull request merged**